storm-server/src/main/java/org/apache/storm/daemon/metrics/reporters/ConsolePreparableReporter.java

```diff
@@ -17,13 +17,17 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
+
+import org.apache.storm.DaemonConfig;
 import org.apache.storm.daemon.metrics.ClientMetricsUtils;
+import org.apache.storm.utils.ObjectReader;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ConsolePreparableReporter implements PreparableReporter {
     private static final Logger LOG = LoggerFactory.getLogger(ConsolePreparableReporter.class);
     ConsoleReporter reporter = null;
+    Integer reportingIntervalPeriod = null;
```

Comment:
It would be better to change `reportingIntervalPeriod` to `reportingIntervalSecs`